packages/web-runtime/src/index.ts

```diff
@@ -115,6 +116,19 @@ export const bootstrapApp = async (configurationPath: string): Promise<void> =>
     }
   )
 
+  store.watch(
+    (state, getters) => {
+      return getters['runtime/auth/isIdpContextReady']
+    },
+    async (idpContextReady) => {
+      if (!idpContextReady) {
+        return
+      }
+      // fake spaces being loaded so that the ApplicationLayout doesn't stay in loading state
+      store.commit('runtime/spaces/SET_SPACES_INITIALIZED', true)
```

Comment:
ughhhhhhhh, maybe we add something else like idpContext and check on that in ApplicationLayout
like that runtime/auth/isIdpContextReady

Reply:
uh, you're right, thanks! In the `Application`-layout I can just add `idp` to the list of authContext types that ignore if spaces are loaded. 🙈  Will do!

Reply:
done